src/kedro_neptune/__init__.py

```diff
@@ -439,6 +475,7 @@ def after_node_run(self, node: Node, catalog: DataCatalog, outputs: Dict[str, An
 
         log_data_catalog_metadata(namespace=run, catalog=catalog)
         run.container.sync()
+        catalog.release("neptune_run")
```

Comment:
What is it?

Reply:
Think about it the same way as leaving a context-manager or `close` an operation. It stops underlying Run object after node execution end.